src/models/tools/geometry/jxg-board.ts

```diff
@@ -57,19 +57,22 @@ export function sortByCreation(board: JXG.Board, ids: string[], reverse: boolean
 }
 
 function combineProperties(domElementID: string, defaults: any, changeProps: any, overrides: any) {
+  const { id, ...otherProps } = changeProps;
+  otherProps.boundingBox = scaleBoundingBoxToElement(domElementID, changeProps);
```

Comment:
I'm not sure the rescaling logic needs to be here at all any more, but I left it in